oga/backend/users/views.py

```diff
@@ -47,32 +82,30 @@ def questions(request):
         response_dict = {'id': question.id}
         return JsonResponse(response_dict, status=201)
 
+@check_request
 @csrf_exempt
+@require_http_methods(["POST"])
 def sign_up(request):
     if request.method == 'POST':
```

Comment:
Actually I left that line to prevent unwanted mistakes. As we can have other methods to use in future implementations. But as you said some functions like signout, signin, signup have only one method POST. I will modifiy them.

Reply:
I see! But since the decorator will catch any errors in request methods I think it should be safe to remove them. If we add more methods in future commits we can simply add them to the require_http_methods decorator too.

Reply:
Oh I forgot to remove that line. thank you